python/src/deltachat/const.py

```diff
@@ -139,7 +147,7 @@
 
 
 def read_event_defines(f):
-    rex = re.compile(r'#define\s+((?:DC_EVENT_|DC_QR|DC_MSG|DC_STATE_|DC_STR|'
+    rex = re.compile(r'#define\s+((?:DC_EVENT_|DC_QR|DC_MSG|DC_LP|DC_STATE_|DC_STR|'
```

Comment:
Why is it `DC_EVENT_` and `DC_STATE_` but `DC_QR`, `DC_MSG` and `DC_LP`? Maybe move `_` outside?